main.rb

```diff
@@ -1,20 +1,63 @@
-require_relative 'person'
-require_relative 'student'
-require_relative 'teacher'
-require_relative 'nameable'
+require_relative 'app'
+def display
+  {
+    1 => 'list_all_books',
+    2 => 'list_all_people',
+    3 => 'create_a_teacher',
+    4 => 'create_a_student',
```

Comment:
- [ ] According to the wireframe, there must be one option 'create a person'. Then you need to ask the user if to create a teacher or a student and get user input that way and return the required methods afterward. Kindly apply it.
![image](https://github.com/vanina123/School-libary/assets/91486791/f86fc57c-2430-4a05-98a2-2953eb379d90)
![image](https://github.com/vanina123/School-libary/assets/91486791/56b26964-448a-4088-b786-0a892021b967)